lib/docker-sync/dependencies/docker_driver.rb

```diff
@@ -5,7 +5,7 @@ module Driver
         def self.docker_for_mac?
           return false unless Environment.mac?
           return @docker_for_mac if defined? @docker_for_mac
-          @docker_for_mac = Environment.system('pgrep -q com.docker.hyperkit')
```

Comment:
Let's add a comment what stands for what `com.docker.hyperkit` for the 'old virtualization' and `com.docker.virtualization` for the new one (maybe a link?)

Reply:
I know the new virtualization framework options from here https://www.docker.com/blog/speed-boost-achievement-unlocked-on-docker-desktop-4-6-for-mac/

But the performance improvement is not good enough for me, so I still need docker-sync for big projects.

Maybe it's better for you to write the comment, English is not my native language.

Reply:
English is not my native tounge either

i would suggest

`Detects new docker-desktop-for-mac virtualization engine, see https://www.docker.com/blog/speed-boost-achievement-unlocked-on-docker-desktop-4-6-for-mac/`

Reply:
I would say IMOH, just use proper semver, and afaik v1.0 is considered equal to 1.0